app/templates/buyers/start_brief_info.html

```diff
@@ -53,7 +53,15 @@
         <h2>Before you start</h2>
         <ol class="list-number">
             <li>You can talk to suppliers to prepare your requirements.<br>
-                <a href="#">View a list of suppliers.</a>
+                <a href="https://assets.digitalmarketplace.service.gov.uk/catalogues/digital-outcomes-and-specialists/{{ lot.slug }}-suppliers.csv">
+                    View a list of suppliers
+                </a> (CSV, {{
```

Comment:
Let's drop these ones for now too. It feels like the files are likely to be updated so the sizes can get out of date pretty quickly. I think we should remove them for now and add S3-based size info later if we want to.